<launch>

    <node name="ros_socket" pkg="ROS_Socket" type="Hiwin_RT605_Socket.py" output="screen"/>

    <node name="ros_strategy" pkg="ROS_Socket" type="Hiwin_RT605_Strategy_test.py" output="screen"/>

</launch>
